source/meta_heritage_frost.rst

```diff
@@ -0,0 +1,46 @@
+Heritage system: Frost
+""""""""""""""""""""""
+
+.. Insert the name of the heritage metadata system in the above heading. No   
+   other text should go under
+   this heading.
+
+
+Responsible(Who?)
+==================
+
+.. Required. Who is responsible for this heritage system. This can be a 
+   group, a role or an administrative unit. Try to avoid linking to specific  
+   persons.
+
+Department of Observation and Climate. Division for Observation quality and data processing.
+
+Description(What?)
+==================
+
+.. Required. Short description of the system: 
+   - what types of metadata is stored in this system.
+   - how is the metadata stored
+   - formats/language
+
+Frost is a RESTful API that provides access to MET Norway's archive of historical weather and climate data.
```

Comment:
but is it all weather and climate data? I thought it was only in-situ observations..?

Reply:
this is still unclear to me. Can you specify?